actions/types.ts

```diff
@@ -182,6 +182,12 @@ export interface InvestigatorCampaignNoteCount {
   custom?: boolean;
 }
 
+export interface ChaosBagResults {
+  currentToken?: ChaosTokenType;
+  drawnTokens: ChaosTokenType[];
+  total: number;
```

Comment:
What's total -- the total # of tokens in the bag / drawn?  Or total available?  I might suggest making this derived state (based on what's drawn + sealed + available).

What behavior do we want to happen when a token is added/removed from the bag (probably nothing right, the sealed tokens stay sealed, drawn tokens stayed drawn, etc).

Reply:
Total should be renamed to something like `totalTokensDrawn`. It really isn't even necessary, just some added flare.

When a token is added/removed from the bag, everything should continue to work just with the new state of the bag.